WeatherForecast.cs

```diff
@@ -1,6 +1,6 @@
 using System;
 
-namespace simulation
+namespace bballsim.league.simulation

```

Comment:
can we get rid of .league in the folder/namespace structure? After battling with the common namespace thing some today, I do not think the league common path will grant any help. Let's just do bballsim.nameOfApp or whatever (i.e., cut out "league")